controls/3.6/cf_serverd.cf

```diff
@@ -69,6 +69,13 @@ bundle server access_rules()
       comment => "Grant access to the policy updates",
       admit => { @(def.acl) };
 
+# JB repository access grant
```

Comment:
I would reccomend defining your custom ACLS in a new policy file to make future framework upgrades easier.

Simply:

my_acl_rules.cf

```
bundle server my_acl_rules
{
  access:
      "$(def.jb_dir_repository)"
         handle => "server_access_grant_access_jb_repository",
         comment => "Grant access to the JB repository",
         admit   => { @(def.acl) };
}
```